src-gui/src/renderer/components/pages/help/ExportDataBox.tsx

```diff
@@ -0,0 +1,85 @@
+import {
+  Box,
+  Typography,
+  makeStyles,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  Button,
+} from "@material-ui/core";
+import InfoBox from "renderer/components/modal/swap/InfoBox";
+import { useState } from "react";
+import { getWalletDescriptor } from "renderer/rpc";
+import { ExportBitcoinWalletResponse } from "models/tauriModel";
+import PromiseInvokeButton from "renderer/components/PromiseInvokeButton";
+import ActionableMonospaceTextBox from "renderer/components/other/ActionableMonospaceTextBox";
+
+const useStyles = makeStyles((theme) => ({
+  content: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "flex-start",
+    gap: theme.spacing(2),
+  }
+}));
+
+export default function ExportDataBox() {
+  const classes = useStyles();
+  const [walletDescriptor, setWalletDescriptor] = useState<ExportBitcoinWalletResponse | null>(null);
+
+  const handleSuccess = (value: ExportBitcoinWalletResponse) => {
+    setWalletDescriptor(value) 
+  }
+
+  const handleCloseDialog = () => {
+    setWalletDescriptor(null);
+  };
+
+  return (
+    <InfoBox
+      title="Export Data"
+      icon={null}
+      loading={false}
+      mainContent={
+        <Box className={classes.content}>
+          <Typography variant="body1">
+            Export your Bitcoin wallet descriptor for backup or recovery purposes.
+            The wallet descriptor is a JSON object that can be used to derive the wallet's private keys.
+            It can be imported into other Bitcoin wallets or services that support the descriptor format.
+            It should thus be stored securely.
+          </Typography>
+          <PromiseInvokeButton
+            variant="outlined"
+            onInvoke={getWalletDescriptor}
+            onSuccess={handleSuccess}
+            displayErrorSnackbar={true}
+          >
+            Export Bitcoin Wallet Descriptor
+          </PromiseInvokeButton>
+        </Box>
+      }
+      additionalContent={
+        <Dialog open={walletDescriptor !== null} onClose={handleCloseDialog} maxWidth="md" fullWidth>
+          <DialogTitle>Bitcoin Wallet Descriptor</DialogTitle>
+          <DialogContent>
+            {walletDescriptor && (
```

Comment:
An explanation on what to do with the key would be good.

https://github.com/UnstoppableSwap/core/blob/master/docs/asb/README.md#exporting-the-bitcoin-wallet-descriptor

Reply:
We should add this to the official docs pages as well. Perhaps we should move that project to this monorepo as well

Reply:
Maybe but that's not too urgent and it having it's own repo makes deployment a bit easier.

Check out the MoneroRecoveryDialog for a similar example.